glide.yaml

```diff
@@ -32,4 +32,6 @@ import:
   version: ~1.1
 - package: github.com/arschles/assert
   version: 6882f85ccdc7c1822b146d1a6b0c2c48f91b5140
-- package: github.com/minio/minio-go
+- package: github.com/docker/distribution
```

Comment:
Docker distribution doesn't create a bucket if it isn't present and they have a regex which prevents us from using the ':' in the path https://github.com/docker/distribution/blob/master/registry/storage/driver/storagedriver.go#L109.
All our folders are <appname>:<gitsha>